.gitlab-ci.yml

```
@@ -393,6 +395,8 @@ workflow:
 
 .on_main:
   - <<: *if_main_branch
+    variables:
+      GO_TEST_SKIP_FLAKE: false
```

gitlab sorry, better with ?
```suggestion
      GO_TEST_SKIP_FLAKE: "false"
```